source/help/submit/index.md

```diff
@@ -50,6 +50,25 @@ We do not accept [dvi, PS, or PDF created
 from TeX/LaTeX source](../faq/whytex.md), and we
 do not accept scanned documents, regardless of format.
 
+
+<span id="newtex"></span>
+## (La)TeX processing changes &mdash; April 2024
```

Comment:
I would propose the section
(La)TeX processing changes &mdash; April 2024
be moved from
/help/submit
to
help/submit_tex
This content seems more relevant on the TeX specific submission page.

Reply:
I made a ticket for this and will take care of it this afternoon. 
https://arxiv-org.atlassian.net/browse/ARXIVCE-1611